### DISCUSSION TITLE
Bugs # 412 - 426 are all design issues - they are not JUST bugs.

### DISCUSSION THREAD

**Discussion by PellTheTabber1** [powertab/powertabeditor#428] [Ideas]
The details to reproduce these bugs are NOT the issue.  The system is simply too prone to bugs in general.  There are just too many bugs n the software to use it with any confidence that the files won't end up.  It almost seems like there is a randomness to the software.  There are so many issues that the scores are all BOUND to wind up mal-functioning.  These issues should be fixed with an eye on design.  I've pretty much had it with the teaching how to use the Pitch wheel, how to use the Model View Controller architecture and whatever else, so please have someone fixing these bugs with a new approach to avoiding them in the first place.  The users can't keep taking chances with the software.  The User community is frustrated.  I've had people message me but I'm not fixing the bugs myself right now so I can't do it.

It might be that i seem to have to enter more rests than seems normal due to some sort of sequencing issue which I truly believe is an error, I can't describe it any other way, other than it seems odd that I have to insert rests all the time to get the midi to sequence properly.  With enough rests, perhaps it will work in general but something seems wrong.  And it would be nice to have a copy and paste that allows for both voices as an option, perhaps this is an enhancement, or perhaps it already exists.  I've never seen a helpfile for the software.

**Comment by cameronwhite** [comment]
This is not a particularly constructive discussion. I've replied to all of the issues requesting more information, steps to reproduce, or example files so that I can investigate the issue, but none of that information has been provided.

**Reply by PellTheTabber1** [reply]
> This is not a particularly constructive discussion. I've replied to all of the issues requesting more information, steps to reproduce, or example files so that I can investigate the issue, but none of that information has been provided.

A user of the software just needs to report bugs.  They don't need to be picky in documenting the bugs.  The bugs are NOT an uncommon occurence.

That it's not constructive is NOT a reflection on the user of the software.  The files are all out of sync all the time.  It's not necessary for someone who reports a bug to act like it's very peculiar and it's NOT necessary to share your own files.

A user of the software simply needs to mention the behavior, that's enough!

I don't want to hear your excuses.

**Comment by cameronwhite** [comment]
Part of reporting a bug is providing enough information for the developer to reproduce and investigate the bug.

This conversation clearly isn't going anywhere productive though, so I'm locking this.
